### PR TITLE
Item data tff88

### PR DESCRIPTION
**Pull request by gearedmonkey**
### Summary

I added item data. Also simplified the responses for certain API calls to remove duplicate code between server and client.

### Checklist

- [x] Did you run `npm test`?
- [x] Did you update/add documentation for new methods or changed functionality?

Finally, is it ready to be reviewed and merged: yes :white_check_mark:

### How to Review

1. Look at the code changes
2. Run the review for client PR: LINK IN PROGRESS
3. Don't merge until client PR is merged!

### Links

Put links to Jira tickets, and anything else, here.

- [Client PR](https://github.com/TheFourFifths/consus-client/pull/34)

### CODE REVIEW

File: src/store/item-store.js [hunk 1]
```diff
@@ -8,17 +8,24 @@ let items = [
     {
         address: 'iGwEZUvfA',
         modelAddress: 'm8y7nEtAe',
-        status: 'AVAILABLE'
+        status: 'AVAILABLE',
```
Comment (mattmahn) [inline src/store/item-store.js]: Why did you add these properties to these items? It's not even all the applicable properties.

File: src/store/model-store.js [hunk 2]
```diff
@@ -12,7 +12,8 @@ let models = [
         isFaulty: false,
         faultDescription: '',
         price: 10.50,
-        count: 20
+        count: 20,
```
Comment (mattmahn) [inline src/store/model-store.js]: Why did you add these properties to these items? It's not even all the applicable properties.

File: src/store/item-store.js (hunk 1 above)
Comment (gearedmonkey) [inline src/store/item-store.js]: What? Items have a status and can be faulty or not.

File: src/store/model-store.js (hunk 2 above)
Comment (gearedmonkey) [inline src/store/model-store.js]: Models have a count in-case someone checks out a wire and models have items?

File: src/store/item-store.js (hunk 1 above)
Comment (mattmahn) [inline src/store/item-store.js]: But why did you only add _some_ of the properties?

Comment (gearedmonkey) [inline src/store/item-store.js]: What properties did I miss?

Comment (mattmahn) [inline src/store/item-store.js]: None. Nevermind, I'm dumb
